pkgs/development/tools/gvls/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, glib
+, vala
+, jsonrpc-glib
+, json-glib
+, gtk3
+, pango
+, gtksourceview3
+, libgee
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gvls";
+  version = "0.14.3";
+
+  outputs = [ "out" "dev" "devdoc" ];
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "esodan";
+    repo = "gvls";
+    rev = "${pname}-${version}";
+    sha256 = "1brz3dxdjfww1b8npzrpzanndmy835w6fvkfydjbqz1h89myv73a";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    pango
+  ];
+
+  propagatedBuildInputs = [
+    glib
+    gtk3
+    gtksourceview3
+    json-glib
+    jsonrpc-glib
+    libgee
+    vala
+  ];
```

Comment:
Do we really want to propagate all of those?

Reply:
Not sure, as I'm inheriting the codebase from worldofpeace. I've only built it and tested the binary. I'll look into it more and come to a conclusion.